package.json

```diff
@@ -41,6 +44,12 @@
     "test-php": "wp-env run phpunit 'phpunit -c /var/www/html/wp-content/plugins/performance/phpunit.xml.dist --verbose'",
     "pretest-php-multisite": "wp-env run composer 'install --no-interaction'",
     "test-php-multisite": "wp-env run phpunit 'WP_MULTISITE=1 phpunit -c /var/www/html/wp-content/plugins/performance/tests/multisite.xml --verbose'",
-    "wp-env": "wp-env"
+    "wp-env": "wp-env",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "*.php": [
+      "composer run-script lint"
```

Comment:
It should be `npm run lint-php`, shouldn't it?

Reply:
Running `npm run lint-php` executes the commands in the docker environment. It is primarily used in Gutenberg for [running unit tests in CI](https://github.com/WordPress/gutenberg/blob/74a99b2cfce30a49d4c007960f24314ba1a1983e/.github/workflows/unit-test.yml#L78).  We were using it earlier, but we faced an issue as discussed [here](https://github.com/WordPress/performance/pull/177#issuecomment-1059080228). That's why we went with `composer run-script lint` as it ran on host.

Reply:
Ok, then, i think, we need to update the `lint-php` command to use `composer run-script lint` for consistency. Right?

Reply:
I don't think it's necessary as we may need to run the linting command in the docker environment in future during unit testing(as done by Gutenberg). 

So I think it's okay to keep it as-is. Similar to what's here, even Gutenberg has different commands in `package.json` and `composer.json`. We don't need to emulate them every time, but I don't see any harm in keeping it as-is.

Reply:
I agree with @kirtangajjar. `npm run lint-php` acts in the Docker container, like most of the other `npm` scripts, so that it's easier to set up and doesn't depend on the local environment. So we shouldn't change that.

Based on the comment https://github.com/WordPress/performance/pull/177#issuecomment-1059080228, it's a fair point to use `composer run-script lint` here, but it feels also like a workaround to me. That may be okay, since linting is basic enough so that it would still work from just running `composer install` and `npm install`.

The main thing here is that we shouldn't require contributors to do anything else in order to use the critical features. For example, `composer run-script test` must never become the only command to run tests, as that would require a lot more to set up on your environment.